pkg/console/subresource/consoleserver/config_builder_test.go

```diff
@@ -421,6 +422,10 @@ func TestConsoleServerCLIConfigBuilder(t *testing.T) {
 								Missing:  []authorizationv1.ResourceAttributes{{Resource: "clusterroles", Verb: "list"}},
 							},
 						},
+						PinnedResources: []configv1.GroupVersionResource{
+							{Group: "apps", Version: "v1", Resource: "Deployment"},
+							{Group: "core", Version: "v1", Resource: "ConfigMap"},
```

Comment:
Do we want use "core" or "" as group? Asking because the enhancement doc uses "" instead:

```suggestion
							{Group: "core", Version: "v1", Resource: "ConfigMap"},
```